diff --git a/src/language_en.h b/src/language_en.h
--- a/src/language_en.h
+++ b/src/language_en.h
@@ -21,5 +21,1 @@
-#ifdef _MSC_VER
-#pragma execution_character_set("utf-8")
-#endif
-
 
diff --git a/src/language_en_gb.h b/src/language_en_gb.h
--- a/src/language_en_gb.h
+++ b/src/language_en_gb.h
@@ -34,5 +34,1 @@
-#ifdef _MSC_VER
-#pragma execution_character_set("utf-8")
-#endif
-

diff --git a/src/language_es.h b/src/language_es.h
--- a/src/language_es.h
+++ b/src/language_es.h
@@ -34,5 +34,1 @@
-#ifdef _MSC_VER
-#pragma execution_character_set("utf-8")
-#endif
-
 
diff --git a/src/language_es_mx.h b/src/language_es_mx.h
--- a/src/language_es_mx.h
+++ b/src/language_es_mx.h
@@ -34,5 +34,1 @@
-#ifdef _MSC_VER
-#pragma execution_character_set("utf-8")
-#endif
-
 
diff --git a/src/language_fr.h b/src/language_fr.h
--- a/src/language_fr.h
+++ b/src/language_fr.h
@@ -34,5 +34,1 @@
-#ifdef _MSC_VER
-#pragma execution_character_set("utf-8")
-#endif
-
 
diff --git a/src/language_zh_cn.h b/src/language_zh_cn.h
--- a/src/language_zh_cn.h
+++ b/src/language_zh_cn.h
@@ -34,5 +34,1 @@
-#ifdef _MSC_VER
-#pragma execution_character_set("utf-8")
-#endif
-
 
